docs/api/public_php_api.md

```diff
@@ -1123,13 +1093,15 @@ $contentCreateStruct->setField( 'image', array(
 
 ### Create Content with XML Text
 
+The XML Text Field Type is not officially supported by eZ Platform. PlatformUI also does not support WYSIWYG editing of Fields of this type.
+
 !!! note "Full code"
 
     <https://github.com/ezsystems/CookbookBundle/blob/master/Command/CreateXMLContentCommand.php>
 
-Another very commonly used Field Type is the rich text one, `XmlText`.
+Another very commonly used Field Type is the RichText one, `XmlText`. 
 
-**working with xml text**
+**Working with xml text**
```

Comment:
That whole section no longer applies to core features, if you check the links you'll see 404. XmlText was replaced by RichText and the latter one is undergoing maintenance now (it's being moved out of Kernel). XmlText is available as a separate dependency: [ezsystems/ezplatform-xmltext-fieldtype](https://github.com/ezsystems/ezplatform-xmltext-fieldtype).